src/Simplex/Chat.hs

```diff
@@ -233,6 +234,10 @@ processChatCommand = \case
     ff <- asks filesFolder
     atomically . writeTVar ff $ Just filesFolder'
     pure CRCmdOk
+  SetIncognito onOff -> do
+    incognito <- asks incognito
+    atomically . writeTVar incognito $ onOff
```

Comment:
```suggestion
    incognito' <- asks incognito
    atomically . writeTVar incognito' $ onOff
```

It usually warns on name shadowing, didn't it? Better to avoid it in any case.